tensorflow/lite/micro/micro_allocation_info.cc

```diff
@@ -316,6 +315,11 @@ TfLiteStatus AllocationInfoBuilder::MarkAllocationLifetimes(
        subgraph->outputs() != nullptr && i < subgraph->outputs()->size(); ++i) {
     const int tensor_index = subgraph->outputs()->Get(i);
     AllocationInfo* current = &subgraph_allocation_info[tensor_index];
+    // Make sure to assign the First created value of the subgraph output
+    // This will handle the case where the subgraph is empty or the output
+    // tensors are unused by the subgraph. This is a sanity measure to
```

Comment:
output tensors unused by the subgraph is not clear to me. Let's remove that comment.